src/js/profile/application_settings.js

```diff
@@ -302,11 +302,14 @@ export class ApplicationSettings extends ReadWriteProxy {
         return 1;
     }
 
-    getMovementSpeed() {
+    /**
+     * @param {boolean} moveFaster
+     */
+    getMovementSpeed(moveFaster = false) {
         const id = this.getAllSettings().movementSpeed;
         for (let i = 0; i < movementSpeeds.length; ++i) {
             if (movementSpeeds[i].id === id) {
-                return movementSpeeds[i].multiplier;
+                return movementSpeeds[i].multiplier * (moveFaster ? 2 : 1);
```

Comment:
Last annotation (Sorry for being so picky again!): I think it doesn't really belong in application_settings - I think it would fit much better in the camera class

Reply:
removed from application_settings and merged it with the camera.js settings